src/CoreWCF.Http/tests/SimpleWSHTTPTest.cs

```diff
@@ -230,14 +227,9 @@ internal void WSHttpRequestImpersonateWithKestrel()
                 ((IChannel)channel).Close();
             }
         }
-#endif
 
-        [Fact]
+        [WindowsNetCoreOnlyFact]
         [Description("transport-security-with-windows-authentication-httpsys")]
```

Comment:
I believe this should be WindowsOnly for the same reasons as previously commented.

Reply:
done